Utils/src/GenLeptonRecoCand.cc

```diff
@@ -344,7 +340,7 @@ GenLeptonRecoCand::produce(edm::StreamID, edm::Event& iEvent, const edm::EventSe
       //      printf("No hadronic tracks!\n");
       selectedTauLeadTrkIso->push_back(-999.);
       selectedTauLeadTrkAct->push_back(-999.);
-      selectedTauLeadTrk->push_back(TLorentzVector(0,0,0,0));
+      selectedTauLeadTrk->push_back(math::PtEtaPhiELorentzVector(0,0,0,0));
```

Comment:
this would be better as `emplace_back(0,0,0,0)`

Reply:
Done